plugins/org.komodo.core/src/config/komodo.cnd

```diff
@@ -56,9 +56,6 @@
  * data sources, and other resources created and being worked on by the user.
  */
 [tko:workspace] > nt:unstructured
-  + tko:repositories (tko:repositories) copy
```

Comment:
With a flat workspace, there seemed no reason to include these. The argument for no grouping nodes for vdbs, models etc.. applies to these objects as well.